py-shared/src/quilt_shared/pkgpush.py

```diff
@@ -46,6 +46,13 @@ class S3ObjectDestination(pydantic.BaseModel):
     bucket: str
     key: str
 
+    @classmethod
+    def from_pk(cls, pk: PhysicalKey):
+        # XXX: do we need this?
```

Comment:
🤷 we can just ignore it

Reply:
well, technically yes, so this check is quite reasonable, tho maybe not very practical. i don't have a strong preference for keeping vs removing this, so use your judgement